src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
-export interface Figure {
+function roundToHundreths(number: number): number {
+  return +(Math.floor(number * 100) / 100).toFixed(2);
```

Comment:
return Math.floor(number * 100) / 100)

If you just leave it like that, it completes same task, without changing to string and back to number ;)